docs/_includes/install/universal.sh

```diff
@@ -9,14 +9,26 @@ if [ "${OS}" = "Linux" ]
 then
     if [ "${ARCH}" = "x86_64" -o "${ARCH}" = "amd64" ]
     then
-        DIR="amd64"
+        # Require at least SSE4.2 (introduced in 2006) but also provide a fallback to SSE2 (introduced in 2000) for older systems. SSE2
+        # builds are much less tested than SSE 4.2 builds. Hardware w/o SSE2 is unsupported.
+        HAS_SSE42=$(grep sse4_2 /proc/cpuinfo)
+        if [ "${HAS_SSE42}" ]
+        then
+            DIR="amd64"
+        else
+            HAS_SSE2=$(grep sse2 /proc/cpuinfo)
```

Comment:
x86_64 has SSE2 by the spec
No need to check cpuinfo.

PS. Some virtual machines provide fake cpuinfo.

Reply:
Thanks, interesting, didn't know that but can confirm.